hpcgap/tst/testinstall/hpc/channels.tst

```diff
@@ -0,0 +1,61 @@
+# Test("../../hpcgap/tst/testinstall/hpc/channels.tst");
```

Comment:
Oops, that line should be removed (and perhaps instead a comment added which explains that the tests below so far are single-threaded only, and more tests should be added which tests channels across threads).